packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleFocusableChildrenGrid.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  Flex,
+  Text,
+  Card,
+  Grid,
+  cardsContainerBehavior,
+  cardChildrenFocusableBehavior,
+  Button,
+  StarIcon,
+  DownloadIcon,
+  MoreIcon,
+} from '@fluentui/react-northstar';
+import { screenReaderContainerStyles } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+type SampleCardProps = {
+  index?: number;
+};
+
+const SampleCard: React.FC<SampleCardProps> = ({ index, ...unhandledProps }) => {
+  return (
+    <Card accessibility={cardChildrenFocusableBehavior} aria-roledescription="user card" {...unhandledProps}>
+      <Card.Header>
+        <Text content={`User #${index}`} weight="bold" />
+      </Card.Header>
+      <Card.Footer>
+        <Flex space="between">
+          <Button content="Action" />
+          <Flex>
+            <Button icon={<StarIcon />} iconOnly text title="Favourite" />
+            <Button icon={<DownloadIcon />} iconOnly text title="Download" />
+            <Button icon={<MoreIcon />} iconOnly text title="More" />
+          </Flex>
+        </Flex>
+      </Card.Footer>
+    </Card>
+  );
+};
+
+const CardExampleFocusableGrid = () => {
+  const navigationMessageId = 'navigationMessage';
+  const defaultMessage = 'Press arrow keys to navigate between cards.';
+  const [navigationMessage, setNavigationMessage] = React.useState(defaultMessage);
+  const timeout = React.useRef<ReturnType<typeof setTimeout>>();
+  const handleGridFocus = () => {
+    console.log(navigationMessage);
+    if (navigationMessage !== null) {
+      timeout.current = setTimeout(() => {
+        setNavigationMessage(null);
+      }, 3000);
+    }
+  };
+
+  const handleGridBlur = e => {
+    if (!e.currentTarget.contains(e.relatedTarget)) {
+      setNavigationMessage(defaultMessage);
+    }
+  };
+
+  const cleanup = () => {
+    if (timeout.current) {
+      clearTimeout(timeout.current);
+    }
+  };
+
+  // make sure our timeout gets cleared if
+  // our consuming component gets unmounted
+  React.useEffect(() => cleanup, []);
```

Comment:
is this working? it looks like it should be called once, at mount, and that's it.

Reply:
I guess the `cleanup` will be triggered at `unmount`?

Reply:
Yep, it should be called on unmout to clean thing up
https://codesandbox.io/s/competent-grass-9ce2m?file=/src/App.tsx